pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+requires = ["setuptools",
+            "setuptools_scm",
+            "wheel",
+            "cython==0.29.14",
+            "jinja2==2.10.3",
```

Comment:
Why the exact pinning here?

Reply:
To ensure that all builds are predictable. It's to replicate the behavior of these files being generated on the machine that builds the sdist.